src/BaseNode.ts

```diff
@@ -1,8 +1,14 @@
 export default class BaseNode {
-    private children: BaseNode[];
+    public static PATH_SIZE = 4;
```

Comment:
Why `4` exactly?

Reply:
`getPathSign` returns a string of 3 chars length plus one more char for leaf or space
https://github.com/fedotov/radix-tree/blob/8e9944a4b8ed9badff32203c6336a0ea318fdc55/src/BaseNode.ts#L47